src/Transports.AspNetCore/IAuthorizationOptions.cs

```diff
@@ -19,6 +19,7 @@ public interface IAuthorizationOptions
     /// for the user within <see cref="HttpContext.User"/>
     /// for at least one role in the list prior to executing the GraphQL request or accepting
     /// the WebSocket connection.  If no roles are specified, authorization is not checked.
+    /// Also implies <see cref="AuthorizationRequired"/> is set to <see langword="true"/>.
```

Comment:
```suggestion
    /// Also implies <see cref="AuthorizationRequired"/> is enabled even if it is set to <see langword="false"/>.
```

It seems to me that such a wording is more understandable.